backend/backend/settings/base.py

```diff
@@ -381,6 +381,9 @@ def get_required_setting(
         "PASSWORD": f"{DB_PASSWORD}",
         "PORT": f"{DB_PORT}",
         "ATOMIC_REQUESTS": ATOMIC_REQUESTS,
+        "OPTIONS": {
+            "application_name": os.environ.get("APPLICATION_NAME", ""),
```

Comment:
@Deepak-Kesavan while running the services locally we'll have to configure these values in the respective `sample.env` files. Shall we add it there as well?
If this env is not set, I hope no error will be thrown

Reply:
@chandrasekharan-zipstack I added a reason why it was not added there, in the PR description. I thought of that and finally thought devs can if the need it in local.

Reply:
If it causes no harm, my suggestion is to add this env. We can treat these `sample.env`s as a registry of env variables that our application uses. We'll lose track of such undocumented envs too.
Also if there's no error / warning thrown we'd most likely not set this env.

If there's a link / doc on how this `APPLICATION_NAME` can help in debugging issues with the DB - that can be added as a comment in the `sample.env` as well

Reply:
@chandrasekharan-zipstack We are merging envs right. Then it should be commented else it might override the value and result in different application name for celery workers.

Reply:
@Deepak-Kesavan Merging env is per service and also will give priority to values in `.env` compared to `sample.env`.
This means for first run value will be taken from `sample.env` and on subsequent runs it will normally prefer existing value in `.env` only.

If this is the case, can we add `APPLICATION_NAME` to `sample.env` too as @chandrasekharan-zipstack mentioned?

Reply:
@Deepak-Kesavan @chandrasekharan-zipstack We can add it to the Contribution guidelines public doc for now, let's see.